style.css

```diff
@@ -0,0 +1,198 @@
+* {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  list-style: none;
+}
+
+body {
+  background-color: #e5e5e5;
+}
+
+.headline {
+  height: 72px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+}
+
+.nav-links {
+  display: none;
+}
+
+.nav-links.open {
+  display: block;
+  position: absolute;
+  top: 0;
+  left: 0;
+  background-color: #6070ff;
+  height: 100%;
+  width: 100%;
+}
+
+.nav-links ul {
+  display: flex;
+}
+
+.nav-links.open ul {
+  display: flex;
+  flex-direction: column;
+  margin-top: 2rem;
+}
+
+.nav-links ul li {
+  margin-right: 10px;
+}
+
+.nav-links.open ul li {
+  margin-top: 1.5rem;
+}
+
+.nav-links ul li a {
+  color: #344563;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.nav-links.open ul li a {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
```

Comment:
- [ ] Kindly make sure your design looks exactly like provided Figma template. Make sure your typography meets the design. In order to use these special google fonts, you must first import them at the top of your CSS file from [google fonts](https://fonts.google.com/) in order to be able to see the changes on your screen.

     _**Example of importing Poppins**_
      `@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');`
     _**and you can use it in CSS like this**_
      `font-family: 'Poppins', sans-serif;`